components/brave_new_tab_ui/containers/newTab/settings/sponsoredImagesToggle.tsx

```diff
@@ -171,7 +180,17 @@ export default function SponsoredImageToggle (
   return (
     <div>
       <ToggleRow>
-        <SettingsText>{getLocale('brandedWallpaperOptIn')}</SettingsText>
+        <VerticalContainer>
+          <SettingsText>{getLocale('brandedWallpaperOptIn')}</SettingsText>
+          <LearnMoreText>
+            {getLocale('sponsoredImageOptInDescription')}
+            {' '}
+            <LearnMoreLink
+              href={loadTimeData.getString('newTabTakeoverLearnMoreLinkUrl')}>
+              {getLocale('sponsoredImageOptInLearnMore')}
+            </LearnMoreLink>
```

Comment:
hey, just wondering if these should be in the same string @mkarolin?

Reply:
There's a just landed util in `$web-common/locale` which should make this pretty easy:

https://sourcegraph.com/github.com/brave/brave-core/-/blob/components/common/locale.ts?L245

In the grdp:
```xml
<message name="IDS_BRAVE_NEW_TAB_SPONSORED_IMAGE_OPT_IN_DESCRIPTION" desc="Description text for setting to opt-in to Sponsored Images on New Tab Page">
  You can earn for seeing Sponsored Images. <ph name="BEGIN_LINK1">$0</ph>Learn more<ph name="END_LINK1">$0</ph>
</message>
```

then to use it:
```tsx
formatLocal('sponsoredImageOptInDescription', {
  $0: content => <a>{content}</a>
})
```

Reply:
I am guessing they are this way to leverage LearnMoreLink, but also it helps to reuse the strings without needing re-translation when they have the same content, but just differ by "Learn more". Like in this case `IDS_BRAVE_NEW_TAB_SPONSORED_IMAGE_NOT_EARNING_TITLE` could match, except for the `.` at the end - so if we can have them either both having `.` or neither, that would help with not having to translate them twice.

Reply:
yeah, `Learn more` is now being reused for 2 different `Sponsored images` description strings here: https://github.com/brave/brave-core/pull/28597/files#diff-5ac0e4f21f572803d00a117564c12b7639220bfe6f43d2e4516033b6a7a0e388R130